### DISCUSSION TITLE
Flaky tests

### DISCUSSION THREAD

**Discussion by Amemecica** [Shelex/cypress-allure-plugin#225] [Q&A]
Is there any way to mark a test as flaky(not the tag, it has to be statusDetail)? Also is there any event listener or bus listener withing the cypress-allure plugin?

**Comment by Shelex** [comment]
Hi @weeknd05 ,
thank you for your question.

> Is there any way to mark a test as flaky(not the tag, it has to be statusDetail)?

When I was testing that 2 years ago it was not working, as flaky/new issue/etc status details icons were handled on allure generate step and my status details were overwritten. Not sure this changed, but worth re-testing again.

> Also is there any event listener or bus listener withing the cypress-allure plugin?

Kind of. There are event listeners for mocha and cypress events which trigger allure reporter methods internally. However you can just subscribe to that events in your code as they are exposed by Cypress.
